### PR TITLE
Consolidate scripts

### PR DESCRIPTION
**Pull request by rubo**
## Changes

- Moved `build/debian` to `scripts/deployment/debian`
- Updated licensing and optimized PPA build script

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a fix or a feature that would cause existing functionality not to work as expected)
- [ ] Documentation update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional or API changes)
- [x] Build-related changes
- [ ] Other: _description_ 

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

### CODE REVIEW

File: scripts/deployment/debian/control [hunk 1]
```diff
@@ -10,6 +10,4 @@ Package: nethermind
 Architecture: amd64
 Multi-Arch: foreign
 Depends: libsnappy-dev, libc6-dev, libc6, unzip, jq, curl, wget
-Description: .NET Core Ethereum client full and actively developed
```
Comment (FalcoXYZ) [inline scripts/deployment/debian/control:13]: This description is made up of two parts. The first one is a synopsis/short description and second part is a longer description.

File: scripts/deployment/publish-ppa.sh [hunk 2]
```diff
@@ -2,29 +2,30 @@
 # SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 # SPDX-License-Identifier: LGPL-3.0-only
 
-#exit when any command fails
-RELEASES="bionic focal jammy kinetic"
-
 set -e
-echo $VERSION > ver
-FIXED_VERSION=$(awk -F. '{ print $1"."$2$3$4"0"}' ver)
```
Comment (FalcoXYZ) [inline scripts/deployment/publish-ppa.sh:10]: You should not remove this, it's there for a reason. PPA versioning is different.

**Review by FalcoXYZ — COMMENTED** [review]
Will review again after the changes :)

File: scripts/deployment/publish-ppa.sh (hunk 2 above)
Comment (rubo) [reply scripts/deployment/publish-ppa.sh:10]: Restored the hack with a comment to remove it with v2.

**Review by FalcoXYZ — APPROVED** [review]
LGTM

**Pull request merged**